c-api/examples/wasm/index.html

```diff
@@ -0,0 +1,24 @@
+<script>
+    Promise.all([
+        fetch('ttfparser.wasm').then(function(x) {
+            return x.arrayBuffer();
+        }).then(function(wasm) {
+            return WebAssembly.instantiate(wasm);
+        }),
+        fetch('TTC.ttc').then(function(x) {
+            return x.arrayBuffer();
+        })
+    ]).then(function(result) {
+        var exports = result[0].instance.exports;
+        var fontBlob = result[1];
+
+        exports.memory.grow(400);
+        // Use heap start as the font address
+        var fontAddress = exports.__heap_base.value + 10000;
+
+        var heapu8 = new Uint8Array(exports.memory.buffer);
+        heapu8.set(fontBlob, fontAddress);
+
+        console.log(exports.ttfp_fonts_in_collection(fontAddress, fontBlob.byteLength));
+    });
+</script>
```

Comment:
New line.

Reply:
A formatter didn't let me… just removed it, thanks :)